src/engine/enginebuffer.h

```diff
@@ -391,8 +393,9 @@ class EngineBuffer : public EngineObject {
     QAtomicInt m_iEnableSyncQueued;
     QAtomicInt m_iSyncModeQueued;
     ControlValueAtomic<QueuedSeek> m_queuedSeek;
-    static constexpr QueuedSeek kNoQueuedSeek = {
-            -1.0, SEEK_NONE}; // value used if no seek is queued
+
+    /// Indicates that no seek is queued
+    static constexpr QueuedSeek kNoQueuedSeek = {mixxx::audio::kInvalidFramePos, SEEK_NONE};
```

Comment:
If this constant is only used in enginebuffer.cpp we could remove it from the header.

Reply:
Unfortunately it's a private class, so it cannot be used from a anonymous namespace. And since it's used for a member variable, the class itself can also not be moved into the anonymous namespace. And making it public also feels wrong. Do you have a suggestion?

Reply:
Then just leave it as is.